src/main.py

```diff
@@ -0,0 +1,24 @@
+import os
+from src.helpers.config import initialize_connections, logger
+from src.transaction_processor import TransactionProcessor
+from src.helpers.database import Database
+from src.helpers.blockchain_data import BlockchainData
+
+
+def main() -> None:
+    chain_name = os.getenv("CHAIN_NAME")
```

Comment:
The PR is not about that but let me just throw it in here anyways. Create an issue out of it if you do not want to address it here.

The convention in the services repo is to use network for the name of the chain. There is also an environment variable `NETWORK` set up containing the chain.
The names for chains currently used are `mainnet`, `xdai`, and `arbitrum-one`.
We should only deviate from that convention if we have a really good reason to do that.